### PR TITLE
feat(initial): Initial setup and build files of ckan

### PR DESCRIPTION
**Pull request by SyeKlu**
in the pull request the initial github actions setup is added as well as the build files for the ckan containers

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* @teutonet/k8s
```
Comment (cwrau) [inline .github/CODEOWNERS:1]: This doesn't make sense

```suggestion
```

**Review by cwrau — CHANGES_REQUESTED** [review]
Why push to docker hub?
I'd rather push to ghcr?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  build:
+    name: "Build ${{ matrix.image.name }}"
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - name: teutonet/ckan
+            path: ckan/ckan
+            platforms: linux/amd64
```
[the remark this answers is not in the document]
Comment (SyeKlu) [reply .github/workflows/build.yml:27]: i have added it to the solr container. unfortunately the base image of the others does not support arm

[the remark this answers is not in the document]
Comment (cwrau) [reply .github/workflows/build.yml:27]: Mh, that's unfortunate, this also kinda prevents the detection of images, as there is no good way to configure the architectures 🤔

**Pull request closed without merging**